README.md

```diff
@@ -1 +1,198 @@
+- **[Tasks_are_pushed_using_javascript]**
+- **[Webpack_is_used]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link]  <a href="#"><button type="button" color:red background: white>Visit Site</button></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+A new developer could make use of this project by cloning the repository and could make adjustments
+in the code accordingly.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+1- Install Visual Studio Code<br>
+2- Install Live server Extension in Visual studio code
+ 
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/zunairkhan811/To-do-list.git
+```
+
+
+### Install
+
+Install this project with:
+
+
+--> Visual Studio Code
+
+
+### Usage
+
+To run the project, execute the following command:
+
+
+--> Click on live Link or Click on live server in Vs Studio.
+
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+In order to run for linter errors, you will have to create a pull request
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+
+--> By purchasing domain and hosting
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHOR -->
+
+## 👥 Author <a name="author"></a>
+
+
+
+👤 **Author**
+
+- GitHub: [@zunairkhan811](https://github.com/zunairkhan811)
+- Twitter: [@zunairkhan811](https://twitter.com/zunairkhan811)
+- LinkedIn: [zunairkhan811](https://linkedin.com/in/zunairkhan811)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Will_try_update_html_file]**
+- [ ] **[Will_try_to_update_css_file]**
+- [ ] **[will_try_to_update_javascript_file]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project kindly follow us and give rating by clicking on star button to this repository.
+Thanks!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to give credit to Microverse for inspiring me enough to make this project.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./license.md) licensed.
```

Comment:
- [ ] Kindly fix the path to the `MIT` License in your readme file a it raises a `404` page error when visited. Also if you have not deployed the project yet, You can indicate that it is not available for now. This will make your readme more professional